fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""Utility class for plotting metrics with error ranges."""
+
+from typing import List, Union
+import pandas as pd
+import numpy as np
+
+from ._metric_frame import MetricFrame
+from matplotlib.lines import Line2D
+from matplotlib.axes._axes import Axes
+
+_GIVEN_BOTH_ERRORS_AND_CONF_INT_ERROR = \
+    "Ambiguous Error Metric. Please only provide one of: errors or conf_intervals."
+_METRIC_AND_ERRORS_NOT_SAME_LENGTH_ERROR = \
+    "The list of metrics and list of errors are not the same length."
+_METRIC_LENGTH_ZERO_ERROR = \
+    "No metrics were provided to plot. A nonzero number of metrics is required."
+_METRIC_AND_CONF_INTERVALS_NOT_SAME_LENGTH_ERROR = \
+    "The list of metrics and list of conf_intervals are not the same length."
+
+_METRICS_NOT_LIST_OR_STR_ERROR = \
+    """Metric should be a string of a single metric or a list of the metrics
+     in provided MetricFrame, but {0} was provided"""
+_ERRORS_MUST_BE_SCALAR_OR_ARRAY = "Calculated errors must be a scalar or a array of length 2"
+_CONF_INTERVALS_MUST_BE_ARRAY = "Calculated conf_intervals must be array of length 2"
+_ERRORS_NEGATIVE_VALUE_ERROR = "Calculated relative errors must be positive"
+_CONF_INTERVALS_FLIPPED_BOUNDS_ERROR = \
+    "Calculated conf_intervals' upper bound cannot be less than lower bound"
+
+
+def _is_arraylike(error):
```

Comment:
`error` seems strange as an argument name. Also, does `numpy` have a function to do this?

Reply:
I don't think so, arraylike in numpy seems to be anything that can be transformed into an np.ndarray, but we don't want that since scalars can also be transformed

Reply:
OK - but can you change the argument name, please?